### PR TITLE
Clean up search and replace pairs when importing

### PR DESCRIPTION
**Pull request by dchymko**
## Description
Removes spaces and other whitespace/non-printable chars from the `--search-replace` pairs when running an import.  This will prevent problems such as cut and paste errors, spaces between pairs( `www.domain1.com, www.domain2.com`) , etc

Note: there is some redundancy in the regex pattern between the low (1-32) chars and the "\s" pattern but I thought it was best to have both. 

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `./dist/bin/vip-import-sql @1000.dev --search-replace="www.dom ain1.com,    www.domain2.com" ./db.sq`
1. Verify that repairs displayed in confirmation are free from extraneous chars

### CODE REVIEW

File: src/lib/search-and-replace.js [hunk 1]
```diff
@@ -150,7 +150,10 @@ export const searchAndReplace = async (
 	}
 
 	// determine all the replacements required
-	const replacementsArr = pairs.map( str => str.split( ',' ) );
+	const replacementsArr = pairs.map( str => {
+		const filtered = str.replace( /[\s\x00-\x1F\x7F]/g, '' ); //whitespace & char codes 0-32, 127.
+		return filtered.split( ',' );
```
Comment (elazzabi) [inline src/lib/search-and-replace.js]: Is this better than using `trim()` directly on the strings? Something like:

```js
replacementsArr.map( str => str.split( ',' ).map( s => s.trim() ) );
```

**Review by elazzabi — DISMISSED** [review]
Left a note, but otherwise looks good.

File: src/lib/search-and-replace.js (hunk 1 above)
Comment (dchymko) [reply src/lib/search-and-replace.js]: Good suggestions. Two reasons I didn't go forward for that:
1) It won't remove spaces or special chars from _within_ the string
2) trim only removes whitespace and not the other non-printables. 

But it feels like I'm overthinking this, so I'll probably refactor a bit along the lines of your suggestion.

Comment (elazzabi) [reply src/lib/search-and-replace.js]: oh, thank you for working on that! Looks good to me

**Review by elazzabi — APPROVED** [review]
Looks good 🎉

**Pull request merged**